CRM/Core/Payment/AuthorizeNetIPN.php

```diff
@@ -159,24 +174,6 @@ public function recur($input, $ids, $objects, $first) {
 
     $now = date('YmdHis');
 
-    //load new contribution object if required.
-    if (!$first) {
-      // create a contribution and then get it processed
-      $contribution = new CRM_Contribute_BAO_Contribution();
-      $contribution->contact_id = $ids['contact'];
-      $contribution->financial_type_id = $objects['contributionType']->id;
-      $contribution->contribution_page_id = $ids['contributionPage'];
-      $contribution->contribution_recur_id = $ids['contributionRecur'];
-      $contribution->receive_date = $input['receive_date'];
-      $contribution->currency = $objects['contribution']->currency;
-      $contribution->payment_instrument_id = $objects['contribution']->payment_instrument_id;
```

Comment:
Due to the test fail I didn't copy this ^^ line over & instead fixed completeOrder to calculate it